pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <groupId>com.meltmedia.jgroups</groupId>
   <artifactId>jgroups-aws</artifactId>
-  <version>1.5.1-SNAPSHOT</version>
+  <version>2.0.0-SNAPSHOT</version>
```

Comment:
If users can move from `1.5.0` to this version, without too much work, then we should probably just move to `1.6.0-SNAPSHOT`.